tslint.json

```diff
@@ -109,6 +109,15 @@
     "component-class-suffix": true,
     "directive-class-suffix": true,
     "templates-use-public": true,
-    "invoke-injectable": true
+    "invoke-injectable": true,
+    "callable-types": true,
+    "import-blacklist": [true, "rxjs"],
+    "import-spacing": true,
+    "interface-over-type-literal": true,
+    "no-empty-interface": true,
+    "no-string-throw": true,
+    "prefer-const": true,
+    "typeof-compare": true,
+    "unified-signatures": true,
```

Comment:
Also ... not expecting you to fix all the lint errors in this PR.  Because `prefer-const` causes a ton of churn!